packages/react-scripts/package.json

```diff
@@ -77,6 +76,7 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "devDependencies": {
+    "@svgr/webpack": "^6.1.2",
```

Comment:
This would still be a dependency of `react-scripts`, as we want it to install when users install `react-scripts` :)

Reply:
You are right. Put it there by inertia.